### ISSUE TITLE
aria-keyshortcuts does not work, output of accesskey is not consistent

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#246]
### Summary

aria-keyshortcuts does not work (Firefox, IE 11), output of accesskey is not consistent

1. save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>accesskey</title>
	</head>
	<body>
		<h1>HTML: accesskey</h1>
		<a href=# accesskey=l>test1</a>
		<button accesskey=b>test2</button>
		<input accesskey=e aria-label=test3>
		
		<h1>ARIA: aria-keyshortcuts</h1>
		<a href=# aria-keyshortcuts=l>test4</a>
		<button aria-keyshortcuts=b>test5</button>
		<input aria-keyshortcuts=e aria-label=test6>

		<span tabindex=0 role=link aria-keyshortcuts=l>test7</span>
		<span tabindex=0 role=button aria-keyshortcuts=b>test8</span>
		<span tabindex=0 role=textbox aria-keyshortcuts=e aria-label=test9>test9</span>
	</body>
</html>```
2. Navigate with Tab key, arrow keys, F, V, overview (INS+F7, INS+F5)

### Expected result

* accesskey and aria-keyshortcuts is output with all navigation methods
* Output is consistent across different browsers
* Output occurs at the end of the element

### Actual result

* aria-keyshortcuts is not output at all in IE and Firefox
* output is not consistent (sometimes before and sometimes after the role, sometimes twice) 

#### Chrome 
* Tab: 
	* Button: twice (before and after role)
	* Link: before role
	* Edit: after role
* element list
	* Button: yes
	* Edit: no
	* Link: yes

#### IE and Firefox 
* only accesskey, no aria-keyshortcuts
* Tab: 
	* Button: twice
	* Link: before role
	* Edit: twice
* element list
	* Button: yes
	* Edit: no
	* Link: yes

### Additional Information

#### JAWS version and build number

JAWS 2019.1906.10
 
#### Operating System and version

Windows 8

#### Browser and version:

Internet Explorer 11.0.9600.19431
Chrome 76.0.3809.100
Firefox 68.0.2

**Comment by stevefaulkner** [comment]
tested with chrome Version 120.0.6099.217 (Official Build) (64-bit) and JAWS 2024.2312.53 inconsistent support for `aria-keyshortcuts` found

[test case](https://codepen.io/stevef/pen/QWogjdg)

for example: is not announced consistently on 

```html
<input type="text" aria-keyshortcuts="e" aria-label="test6">
```

virtual navigation using F key its not announced

tab key is announced, but after all other announcements, should be same as `accesskey` after `label` but before `role`

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 68619
